velox/exec/Spill.cpp

```diff
@@ -254,6 +254,24 @@ std::vector<std::string> SpillState::testingSpilledFilePaths() const {
   return spilledFiles;
 }
 
+std::vector<std::unique_ptr<SpillPartition>> SpillPartition::split(
+    int numSplits) {
+  const int32_t numSpiltFiles = bits::roundUp(files_.size(), numSplits);
+  std::vector<std::unique_ptr<SpillPartition>> splits(numSplits);
+
+  for (int split = 0, fileIdx = 0; split < numSplits; ++split) {
+    SpillFiles splitFiles;
+    splitFiles.reserve(numSpiltFiles);
+    while (splitFiles.size() < numSpiltFiles && fileIdx < files_.size()) {
+      splitFiles.push_back(std::move(files_[fileIdx++]));
+    }
+    splits[split] =
+        std::make_unique<SpillPartition>(id_, std::move(splitFiles));
```

Comment:
Why is it ok to share the ID between multiple SpillPartitions? Would you update the documentation for the SpillPartition class to explain that ID is not unique?